### PR TITLE
Improvement: Bash cheatsheet

### PR DESCRIPTION
**Pull request by SwathiSBhat**
## Type of Change

- [ ] New Instant Answer
    - [ ] Cheat Sheet
- [x] Improvement
    - [ ] Bug fix
    - [x] Enhancement
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.)

## Description of new Instant Answer, or changes
Added new contents

---

IA Page: https://duck.co/ia/view/bash_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@SwathiSBhat Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by amarlearning** [comment]
@SwathiSBhat looks like there are two files in your PR, can you please delete the file `Bash.json`.

File: share/goodie/cheat_sheets/json/bash.json [hunk 1]
```diff
@@ -173,6 +176,65 @@
...
+            }
+        ],
+        "Network Configuration and Troubleshooting": [
+          {
+              "key": "ifconfig",
+              "val": "Configure a network interface"
+          },
+          {
+              "key": "ifdown",
+              "val": "Stop a network interface "
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/bash.json]: remove that extra space from end!

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/bash.json]: @SwathiSBhat Please update this one...

**Comment by gautamkrishnar** [comment]
@synbit need your suggestions...

**Comment by sahildua2305** [comment]
@gautamkrishnar please squash all commits into one while merging this one 😉

**Comment by synbit** [comment]
I think that the `ifup` and `ifdown` commands as well as the `less` and `uniq`, even though very useful, would be better suited in a linux cheat sheet or similar. I believe the intention behind bash cheat sheet was to show some basic shell shortcuts (move to the end of line etc...) as well as give some basic examples with regards to the language (i.e. loops, redirections, special variables etc). Given that these basic things are already a lot more than anticipated, I would be tempted to only add content that belongs in the categories mentioned above.

@gautamkrishnar what are your thoughts on this? We can always ask for more opinions and do whatever seems more sensible to the majority.

**Comment by mintsoft** [comment]
> I believe the intention behind the bash cheat sheet was to show some basic shell shortcuts (move to the end of line etc...) as well as give some basic examples with regards to the language (i.e. loops, redirections, special variables etc)

Exactly; this is _not_ a Linux utilities cheat sheet.

**Comment by gautamkrishnar** [comment]
Yes I agree with @ynbit.  @SwathiSBhat please update the code

**Comment by gautamkrishnar** [comment]
@SwathiSBhat we can't add these information the cheatsheet since it is a Bash cheatsheet. 

If you are interested,  please do contribute to the Linux cheatsheethttps://duck.co/ia/view/linux_cheat_sheet

**Pull request closed without merging**